### PR TITLE
.NET 7 Upgrade

### PR DESCRIPTION
**Pull request by aahendry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This repository contains dotnet samples for integration with the Lenus Health pl
 
 ## Requirements
 
-- .NET core 3.1 SDK
+- .NET core 7.0 SDK
 - Visual Studio 2019 or Visual Studio Code
```
Comment (alexszym) [inline README.md]: Can you bump it to 2022 so we don't sound like dinosaurs? 😂

**Review by alexszym — COMMENTED** [review]
Looks good! Only one minor comment about bumping Visual Studio to 2022 from 2019

File: README.md (hunk 1 above)
Comment (aahendry) [reply README.md]: Done.

**Review by alexszym — APPROVED** [review]
Looks good!

**Pull request merged**